### ISSUE TITLE
test broken on Strawberry 5.12.x

### ISSUE THREAD

**Issue by plicease** [plicease/Test2-Plugin-FauxHomeDir#1]
http://www.cpantesters.org/cpan/report/5fb5c17c-6c06-1014-8e5c-cbbf17da2fe0

Probably because bsd_glob doesn't handle `~` correctly on Windows Perl 5.12. 
```

C:\strawberry\perl\bin\perl.exe "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib\lib', 'blib\arch')" t/*.t
# 
# 
# 
# HARNESS_ACTIVE=1
# HARNESS_VERSION=3.32
# PERL5LIB=C:\strawberry\cpan\build\Test2-Plugin-FauxHomeDir-0.04-gKzEc6\blib\lib;C:\strawberry\cpan\build\Test2-Plugin-FauxHomeDir-0.04-gKzEc6\blib\arch;C:\strawberry\cpan\build\Test2-Suite-0.000114-eFexpX/blib/arch;C:\strawberry\cpan\build\Test2-Suite-0.000114-eFexpX/blib/lib;C:\strawberry\cpan\build\Term-Table-0.012-F0ATM3/blib/arch;C:\strawberry\cpan\build\Term-Table-0.012-F0ATM3/blib/lib;C:\strawberry\cpan\build\Sub-Info-0.002-eDZLqN/blib/arch;C:\strawberry\cpan\build\Sub-Info-0.002-eDZLqN/blib/lib;C:\strawberry\cpan\build\Importer-0.025-toplin/blib/arch;C:\strawberry\cpan\build\Importer-0.025-toplin/blib/lib;C:\strawberry\cpan\build\Test-Simple-1.302136-6iqMzS/blib/arch;C:\strawberry\cpan\build\Test-Simple-1.302136-6iqMzS/blib/lib;C:\strawberry\cpan\build\Test2-Suite-0.000114-eFexpX/blib/arch;C:\strawberry\cpan\build\Test2-Suite-0.000114-eFexpX/blib/lib;C:\strawberry\cpan\build\Term-Table-0.012-F0ATM3/blib/arch;C:\strawberry\cpan\build\Term-Table-0.012-F0ATM3/blib/lib;C:\strawberry\cpan\build\Sub-Info-0.002-eDZLqN/blib/arch;C:\strawberry\cpan\build\Sub-Info-0.002-eDZLqN/blib/lib;C:\strawberry\cpan\build\Importer-0.025-toplin/blib/arch;C:\strawberry\cpan\build\Importer-0.025-toplin/blib/lib;C:\strawberry\cpan\build\Test-Simple-1.302136-6iqMzS/blib/arch;C:\strawberry\cpan\build\Test-Simple-1.302136-6iqMzS/blib/lib
# PERL5_CPANPLUS_IS_RUNNING=1512
# PERL5_CPAN_IS_RUNNING=1512
# PERL5_CPAN_IS_RUNNING_IN_RECURSION=376,1512
# PERL_CR_SMOKER_CURRENT=Test2-Plugin-FauxHomeDir-0.04
# PERL_CR_SMOKER_RUNONCE=1
# PERL_EXTUTILS_AUTOINSTALL=--defaultdeps
# PERL_MM_USE_DEFAULT=1
# 
# 
# 
# PERL5LIB path
# C:\strawberry\cpan\build\Test2-Plugin-FauxHomeDir-0.04-gKzEc6\blib\lib
# C:\strawberry\cpan\build\Test2-Plugin-FauxHomeDir-0.04-gKzEc6\blib\arch
# C:\strawberry\cpan\build\Test2-Suite-0.000114-eFexpX/blib/arch
# C:\strawberry\cpan\build\Test2-Suite-0.000114-eFexpX/blib/lib
# C:\strawberry\cpan\build\Term-Table-0.012-F0ATM3/blib/arch
# C:\strawberry\cpan\build\Term-Table-0.012-F0ATM3/blib/lib
# C:\strawberry\cpan\build\Sub-Info-0.002-eDZLqN/blib/arch
# C:\strawberry\cpan\build\Sub-Info-0.002-eDZLqN/blib/lib
# C:\strawberry\cpan\build\Importer-0.025-toplin/blib/arch
# C:\strawberry\cpan\build\Importer-0.025-toplin/blib/lib
# C:\strawberry\cpan\build\Test-Simple-1.302136-6iqMzS/blib/arch
# C:\strawberry\cpan\build\Test-Simple-1.302136-6iqMzS/blib/lib
# C:\strawberry\cpan\build\Test2-Suite-0.000114-eFexpX/blib/arch
# C:\strawberry\cpan\build\Test2-Suite-0.000114-eFexpX/blib/lib
# C:\strawberry\cpan\build\Term-Table-0.012-F0ATM3/blib/arch
# C:\strawberry\cpan\build\Term-Table-0.012-F0ATM3/blib/lib
# C:\strawberry\cpan\build\Sub-Info-0.002-eDZLqN/blib/arch
# C:\strawberry\cpan\build\Sub-Info-0.002-eDZLqN/blib/lib
# C:\strawberry\cpan\build\Importer-0.025-toplin/blib/arch
# C:\strawberry\cpan\build\Importer-0.025-toplin/blib/lib
# C:\strawberry\cpan\build\Test-Simple-1.302136-6iqMzS/blib/arch
# C:\strawberry\cpan\build\Test-Simple-1.302136-6iqMzS/blib/lib
# 
# 
# 
# perl                5.012002
# ExtUtils::MakeMaker 6.98
# Test2::API          1.302136
# Test2::V0           0.000114
# 
# 
# 
#           Before:
#         HOME = undef
#    HOMEDRIVE = C:
#     HOMEPATH = \Documents and Settings\c
#         USER = undef
#     USERNAME = c
#  USERPROFILE = C:\Documents and Settings\c
# 
#           After:
#         HOME = undef
#    HOMEDRIVE = C:
#     HOMEPATH = \DOCUME~1\c\LOCALS~1\Temp\YMIxBjSGF7\home\c
#         USER = undef
#     USERNAME = c
#  USERPROFILE = C:\DOCUME~1\c\LOCALS~1\Temp\YMIxBjSGF7\home\c
# 
# 
# 
t/00_diag.t ........................................... ok
t/01_use.t ............................................ ok
readline() on closed filehandle at t/test2_plugin_fauxhomedir.t line 32.
    # Failed test at t/test2_plugin_fauxhomedir.t line 29.
    # Failed test at t/test2_plugin_fauxhomedir.t line 34.
    # +---------+-------+
    # | GOT     | CHECK |
    # +---------+-------+
    # | <UNDEF> | xx\n  |
    # +---------+-------+

# Failed test 'File::Glob'
# at t/test2_plugin_fauxhomedir.t line 36.
    # Failed test at t/test2_plugin_fauxhomedir.t line 45.

# Failed test 'Path::Tiny'
# at t/test2_plugin_fauxhomedir.t line 48.
# Caught exception in subtest: Error open (<) on '~/test.txt': No such file or directory at t/test2_plugin_fauxhomedir.t line 46.
t/test2_plugin_fauxhomedir.t .......................... 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/6 subtests 
t/test2_plugin_fauxhomedir__with_file_homedir_test.t .. ok

Test Summary Report
-------------------
t/test2_plugin_fauxhomedir.t                        (Wstat: 512 Tests: 6 Failed: 2)
  Failed tests:  3-4
  Non-zero exit status: 2
Files=4, Tests=9,  2 wallclock secs ( 0.07 usr +  0.01 sys =  0.08 CPU)
Result: FAIL
Failed 1/4 test programs. 2/9 subtests failed.
dmake.EXE:  Error code 255, while making 'test_dynamic'
```

**Comment by plicease** [comment]
Hard to argue that anyone using this version of Perl shouldn't upgrade anyway.

**Comment by plicease** [comment]
But patches welcome.

**Comment by plicease** [comment]
should be fixed in `0.06`
